intro_to_R_tidyverse/02-intro_to_tidyverse.Rmd

```diff
@@ -165,14 +162,14 @@ What if we wanted filter this dataset to males who are older than 100 months?
 Let's add another filtering argument to this filter.
 
 ```{r }
-metadata %>% filter(age_mths > 100, sex == "M")
+metadata %>% filter(age_mths > 100, sex == "F")
```

Comment:
The blocks at lines 96 and 100 caused confusion yesterday -- I would suggest either taking them out or making them R code blocks